src/js/views/forms/form/form.scss

```diff
@@ -189,6 +189,30 @@
   font-size: 12px;
 }
 
+.form__locked {
```

Comment:
I think this can be generalized with the css above.  Something like `form__status` as it seems like the design here will always be `| [icon] some text` so rather than having styles that describe what it does for business logic, it should describe what is does in the design.